cli.js

```diff
@@ -39,7 +39,7 @@ try {
 		cwd: cli.flags.cwd || process.cwd(),
 		rename: cli.flags.rename,
 		parents: cli.flags.parents,
-		overwrite: cli.flags.overwrite
+		overwrite: !cli.flags.noOverwrite
```

Comment:
@sindresorhus didn't know that! But still `!cli.flags.noOverwrite` looks better than `cli.flags.overwrite === undefined ? true : cli.flags.overwrite` - should I change it?